### ISSUE TITLE
[Bug Report][3.1.10] v-label (v-checkbox, v-switch, etc) doesn't pass WCAG check

### ISSUE THREAD

**Issue by kappaflow** [vuetifyjs/vuetify#16928]
### Environment
**Vuetify Version:** 3.1.10
**Vue Version:** 3.2.47
**Browsers:** Chrome 110.0.0.0
**OS:** Windows 10

### Steps to reproduce
Add v-checkbox, v-switch, etc with a label, check contrast taking into account the opacity

### Expected Behavior
Probably it should have `opacity: var(--v-high-emphasis-opacity)`

### Actual Behavior
Currently v-label has `opacity: var(--v-medium-emphasis-opacity);`, so the contrast is 3.27:1 (for a light theme text-color:#8E8E8E, background-color:#8E8E8E)

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVUttO5DAM/RUrLwPStIW9aFezZcVqPwHEC+Eh03pomCaNknQAjfrvOBcKDAKJPLQ59rGP7fh6z3Yjerl5LO8cWzGpzGA97KGxKDxeJR9MsLGDgkXmLrjmGh8itRm085AdcPY28GjPNYDvUOEK4h2gxY0Ye3+ZjItW2O1iGVwTfafjP1yzJftnTEk5qaLaNVYa/zcwsmTOMCcUXsDR8TMEsOhHq18wgOtli3YFpycnUSqcKV2CagR1NSvVHpXpqY2gWrdyB7I940wYw1kwcV/vCkKxKDqEXIdIw+iFc8Q0oviZqOkQoemw2a6HB+jFGnvi/M8G4tXViz8GcR8l3L30TTdHXESY+clH7MgNhlBBxAGl6uqKio//Vx0RTJ2Co0El2tw6zT7tQKGEoZ0YNL1BnCTPDsfZ/JY8bE/AnHXeG7eqqlGb7W3ZDKo6J19lR+2lwqId1Pn38lv54xeV5Pxre4lOFWs73Du0pMhZfqKYPGzRRwLZTXlPy98pbTYVNDIX8r7LV5HIDm1hUdNGoP20+APumwYOfO+ayFs10UB7qbfuYJaNi3O8fi7t693FDCH8JqhMS5aqI70YwW6eALHzSiA=)

### Other comments
Related to #16912

**Comment by davycardinaal** [comment]
+1 on this, I also came across this issue.

**Comment by crystalfp** [comment]
For the `v-expansion-panel-title` seems the text-shadow is hardcoded in the button styling to `#111111`. Changing this color to `#494949` (or changing the opacity) will makes the titles readable in light mode. Instead for `v-tabs` buttons seems the shadow color should be lighter, due to the white background. I tried with `#898989`. 
All in all, vuetify with dark theme looks great, with light theme so and so.
BTW, I'm not using any blueprint.
